array_api_strict/_utility_functions.py

```diff
@@ -57,7 +56,7 @@ def diff(
     # currently specified.
 
     # NumPy does not support prepend=None or append=None
-    kwargs = dict(axis=axis, n=n)
+    kwargs: dict[str, Any] = dict(axis=axis, n=n)
```

Comment:
The cool kids are all constructing their dicts as `{...}` instead of `dict(...)` nowadays, so how about:

```suggestion
    kwargs: dict[str, int | npt.NDArray[Any]] = {"axis": axis, "n", n}
```

I sneakily narrowed the dict type as well, which should get you couple of extra swag-creds